python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/priority-flow-control.j2

```diff
@@ -15,12 +15,12 @@ priority-flow-control pause watchdog action {{ priority_flow_control.watchdog.ac
 {%     if priority_flow_control.watchdog.timeout is arista.avd.defined %}
 priority-flow-control pause watchdog default timeout {{ priority_flow_control.watchdog.timeout }}
```

Comment:
watchdog action  command will come after the polling-interval

Reply:
Fixed.